src/Documentation/doc/advanced/corporation/financial-statement.md

```diff
@@ -0,0 +1,163 @@
+
+## Investment offer
+
+There are 4 investment rounds.
+
+Each round has its own `FundingRoundShares` and `FundingRoundMultiplier`.
+
+- $FundingRoundShares = [0.1, 0.35, 0.25, 0.2]$
+- $FundingRoundMultiplier = [3, 2, 2, 1.5]$
+
+Formula:
+
+$$Offer = CorporationValuation\ast FundingRoundShares\ast FundingRoundMultiplier$$
+
+Analyses:
+
+- Offer depends on `Funds`, `AssetDelta` and `NumberOfOfficesAndWarehouses`.
+  - `Funds` are usually depleted to improve divisions.
+  - `NumberOfOfficesAndWarehouses` is the exponent of the multiplier, it can be increased by creating [dummy division](./miscellany.md). It is an easy way to get higher offer in round 3+, when we have enough funds to do that.
+  - `AssetDelta` is multiplied by 315000, so it is the main source of offer.
+- Assuming that we can sell all produced units and not buy more boost materials, `AssetDelta` is delta of funds, and delta of funds is delta of profit. This is why we try our best to improve profit.
+
+## Dividend
+
+`DividendTax` depends on `CorporationSoftcap`. In BN3, `CorporationSoftcap` is 1.
+
+$$DividendTax = 1 - CorporationSoftcap + 0.15$$
+
+`ShadyAccounting` reduces `DividendTax` by 0.05.
+
+`GovernmentPartnership` reduces `DividendTax` by 0.1.
+
+Formula:
+
+$$TotalDividends = DividendRate\ast(Revenue - Expenses)\ast 10$$
+
+$$Dividend = \left(OwnedShares\ast\frac{TotalDividends}{TotalShares}\right)^{1 - DividendTax}$$
+
+Retained earning:
+
+$$RetainedEarning = (1 - DividendRate)\ast(Revenue - Expenses)\ast 10$$
+
+Dividend is added to player's money. Retained earning is added to corporation's funds. This means if we increase `DividendRate`, corporation's valuation is dwindled.
+
+## Shares
+
+Self-fund:
+
+- Cost 150b.
+- Total shares: 1b.
+- Initial owned shares: 1b.
+
+Use seed money:
+
+- Does not cost money.
+- Total shares: 1.5b.
+- Initial owned shares: 1b.
+
+In each investment round, investors take a percentage of initial owned shares. The percentage of each round is in `FundingRoundShares`.
+
+If your corporation is self-funded and you sell CEO position, you only need 50b to create next corporation.
+
+`TargetSharePrice`:
+
+$$TargetSharePrice = \frac{CorporationValuation*\left( 0.5 + \sqrt{\frac{OwnedShares}{TotalShares}} \right)}{TotalShares}$$
+
+When corporation goes public, the initial share price is `TargetSharePrice`.
+
+Share price is updated in START state.
+
+$$SharePrice = \begin{cases} SharePrice\ast(1 + Math.random()\ast 0.01), & SharePrice \leq TargetSharePrice \newline SharePrice\ast(1 - Math.random()\ast 0.01), & SharePrice > TargetSharePrice\end{cases}$$
+
+Minimum share price is 0.01.
+
+Issue new shares:
+
+- Maximum number of new shares is 20% of total shares.
+- The number of new shares issued must be a multiple of 10 million.
+- New share price: $NewSharePrice = \frac{CorporationValuation\ast\left(0.5 + \sqrt{\frac{OwnedShares}{TotalShares + NewShares}}\right)}{TotalShares}$
```

Comment:
In this formula, and also one of the ones near the bottom, the words in the fraction inside the square-root get *very* small.

I'm not entirely sure what you can do about this... Break it out into a full math block? Use a diagonal slash instead of `\frac` for the division by TotalShares?